src/bundle.js

```diff
@@ -49,22 +49,30 @@ module.exports = async (entryFile, dest, options = {}, log = () => {}) => {
 
   // set defaults, but allow override by passed in values
   const parcelBundleOptions = {
-    cache: true,
-    outDir: dest,
-    contentHash: true, // false if dev, true if prod ??
-    watch: false, // currently false if dev true if prod,
-    minify: false,
-    logLevel: 1,
+    entries: entryFile,
+    defaultConfig: require.resolve('@parcel/config-default'),
+    shouldDisableCache: false,
+    targets: {
+      action: {
+        includeNodeModules: true,
+        distDir: dest
+      }
+    },
+    defaultTargetOptions: {
+      distDir: dest,
+      shouldOptimize: false
+    },
+    shouldContentHash: true,
+    logLevel: 'error',
     ...options
   }
 
   aioLogger.debug(`bundle bundleOptions: ${JSON.stringify(parcelBundleOptions, null, 2)}`)
   log(`bundling ${entryFile}`)
-  const bundler = new Bundler(entryFile, parcelBundleOptions)
+  const bundler = new Bundler(parcelBundleOptions)
 
   const cleanup = async () => {
     aioLogger.debug('cleanup bundler...')
-    await bundler.stop()
```

Comment:
I guess this needs to be replaced with onBuildEnd event from the bundler for the cleanup.

Reply:
For the onBuildEnd of Parcel V1, we have reporter plugin in V2. But i think it's different from the stop API. This event handler is to see what assets were built but not for cleanup.